include/openthread/thread_ftd.h

```diff
@@ -739,6 +739,40 @@ void otThreadRegisterNeighborTableCallback(otInstance *aInstance, otNeighborTabl
  */
 void otThreadSetCcmEnabled(otInstance *aInstance, bool aEnabled);
 
+/**
+ * This function gets the range of router IDs that are allowed to assign to nodes within the thread network.
+ *
+ * @note This API requires `OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE`, and is only used for test purpose. All the
+ * router IDS in the range [aMinRouterIdAllowed, aMaxRouterIdAllowed] are allowed.
+ *
+ * @param[in]   aInstance            A pointer to an OpenThread instance.
+ * @param[out]  aMinRouterIdAllowed  The minimum router ID allowed.
+ * @param[out]  aMaxRouterIdAllowed  The maximum router ID allowed.
+ *
+ * @sa otThreadSetAllowedRouterIdRange
+ *
+ */
+void otThreadGetAllowedRouterIdRange(otInstance *aInstance, uint8_t *aMinRouterIdAllowed, uint8_t *aMaxRouterIdAllowed);
```

Comment:
nit: slightly less verbose?
```suggestion
void otThreadGetRouterIdRange(otInstance *aInstance, uint8_t *aMinRouterIdAllowed, uint8_t *aMaxRouterIdAllowed);
```